sdk/nodejs/x/automation/stack.ts

```diff
@@ -262,15 +321,46 @@ export class Stack {
         }
 
         args.push("--exec-kind", kind);
+
+        // Set up event log tailing
+        const logFile = createLogFile("preview");
+        args.push("--event-log", logFile);
+        const eventLog: EngineEvent[] = [];
+        const logPromise = this.readLines(logFile, (event) => {
+            eventLog.push(event);
```

Comment:
Do we want/need to fill this array up in memory?  If I'm following the code below, we are going to ignore almost all of these later?  Can we just only store the ones we need here - or even just store a `summaryEvent` if/when we see it?

Reply:
Ah yeah good catch, I did clean that up in the Go implementation but missed it here.